core/pom.xml

```diff
@@ -19,19 +19,27 @@
         <!-- because this module requires transitive jakarta.jms.api -->
         <revapi.checkDependencies>true</revapi.checkDependencies>
         <formatterConfigPath>../formatter.xml</formatterConfigPath>
+
+        <mule.version>${project.version}</mule.version>
     </properties>
 
     <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-site-plugin</artifactId>
-                <configuration>
-                    <skip>true</skip>
-                    <skipDeploy>true</skipDeploy>
-                </configuration>
-            </plugin>
-        </plugins>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
```

Comment:
use a different folder for filtering resources, to prevent accidental repmacements on the base resources

Reply:
Rodro I have try some tests and the directory tag is a required one. It sets where the resources folders is going to be present so we should maintain the default "src/main/resources". 

In the include I have moved the properties file to a new folder product-version.

Reply:
why not create a `src/main/resources-filtering` for this?

Reply:
I have done what you suggested, maintain the original src/main/resources without filtering and add a new src/main/resources-filtering with the filter in true and it worked fine.